static/deploy/python/infer.py

```diff
@@ -510,7 +521,7 @@ def predict_video(detector, camera_id):
     fps = 30
     width = int(capture.get(cv2.CAP_PROP_FRAME_WIDTH))
     height = int(capture.get(cv2.CAP_PROP_FRAME_HEIGHT))
-    fourcc = cv2.VideoWriter_fourcc(*'mp4v')
+    fourcc = cv2.VideoWriter_fourcc(* 'mp4v')
```

Comment:
Spaces here should be removed. (`* 'mp4v'` --> `*'mp4v'`)

Reply:
Oh, I can see that pre-commit fixed it like this, I'm gonna delete it.